dotcom-rendering/src/components/ArticleTitle.stories.tsx

```diff
@@ -1,616 +1,246 @@
-							id: '',
-							title: 'Match liveblog title',
-							type: 'Blog',
-						},
-					]}
-					isMatch={true}
-				/>
-			</div>
-		</Wrapper>
-	);
-};
-LiveblogTitle.storyName = 'Liveblog title';
-LiveblogTitle.decorators = [
-	splitTheme([
-		{
+export const LiveBlogDesignWithBlogTag = {
+	args: {
+		...StandardDesignWithBlogTag.args,
+		format: {
 			display: ArticleDisplay.Standard,
 			theme: Pillar.Sport,
 			design: ArticleDesign.LiveBlog,
 		},
-	]),
-];
-
-export const ArticleOpinionTag = () => {
-	return (
-		<Wrapper>
-			<ArticleTitle
-				{...FEArticle}
-				format={{
-					display: ArticleDisplay.Standard,
-					theme: Pillar.Sport,
-					design: ArticleDesign.Standard,
-				}}
-				tags={[
-					{
-						id: '',
-						title: 'Opinion title',
-						type: 'Opinion',
-					},
-				]}
-			/>
-		</Wrapper>
-	);
-};
-ArticleOpinionTag.storyName = 'Article with a Opinion tag';
-ArticleOpinionTag.decorators = [
-	splitTheme([
-		{
-			display: ArticleDisplay.Standard,
-			theme: Pillar.Sport,
-			design: ArticleDesign.Standard,
-		},
-	]),
-];
-
-export const ArticleSeriesTag = () => {
-	return (
-		<Wrapper>
-			<ArticleTitle
-				{...FEArticle}
-				format={{
-					display: ArticleDisplay.Standard,
-					theme: Pillar.Sport,
-					design: ArticleDesign.Standard,
-				}}
-				tags={[
-					{
-						id: '',
-						title: 'Series title',
-						type: 'Series',
-					},
-				]}
-			/>
-		</Wrapper>
-	);
-};
-ArticleSeriesTag.storyName = 'Article with a Series tag';
-ArticleSeriesTag.decorators = [
-	splitTheme([
-		{
-			display: ArticleDisplay.Standard,
-			theme: Pillar.Sport,
-			design: ArticleDesign.Standard,
-		},
-	]),
-];
-
-export const SpecialReportTitle = () => {
-	return (
-		<Wrapper>
-			<ArticleTitle
-				{...FEArticle}
-				format={{
-					display: ArticleDisplay.Standard,
-					theme: ArticleSpecial.SpecialReport,
-					design: ArticleDesign.Standard,
-				}}
-				tags={[
-					{
-						id: '',
-						title: 'Special',
-						type: 'Series',
-					},
-				]}
-			/>
-		</Wrapper>
-	);
-};
-SpecialReportTitle.storyName = 'Special report';
-SpecialReportTitle.decorators = [
-	splitTheme([
-		{
-			display: ArticleDisplay.Standard,
-			theme: ArticleSpecial.SpecialReport,
-			design: ArticleDesign.Standard,
-		},
-	]),
-];
-
-export const SpecialReportAlt = () => {
-	return (
-		<Wrapper>
-			<ArticleTitle
-				{...FEArticle}
-				format={{
-					display: ArticleDisplay.Standard,
-					theme: ArticleSpecial.SpecialReportAlt,
-					design: ArticleDesign.Standard,
-				}}
-				tags={[
-					{
-						id: '',
-						title: 'Special Report Alt',
-						type: 'Series',
-					},
-				]}
-			/>
-		</Wrapper>
-	);
-};
-SpecialReportAlt.storyName = 'Special report Alt';
-SpecialReportAlt.decorators = [
-	splitTheme([
-		{
-			display: ArticleDisplay.Standard,
-			theme: ArticleSpecial.SpecialReportAlt,
-			design: ArticleDesign.Standard,
+	},
+	parameters: {
+		colourSchemeBackground: {
+			light: '#005689',
+			dark: '#004e7c',
```

Comment:
Why did you use this value? I can't see it in the previous stories. This is how it looks like now, just checking this is what you want.

![image](https://github.com/user-attachments/assets/88ca2568-981c-48dd-a95f-f086590ded71)

Reply:
Good spot! But yes, this is intended. The previous stories used `inherit` for this value:

https://github.com/guardian/dotcom-rendering/blob/ffce9fda6710ea915d59b42ac16a09574aa36d29/dotcom-rendering/src/components/ArticleTitle.stories.tsx#L241-L243

which resulted in `#000` being used for the dark mode background:

https://github.com/guardian/dotcom-rendering/blob/ffce9fda6710ea915d59b42ac16a09574aa36d29/dotcom-rendering/.storybook/decorators/splitThemeDecorator.tsx#L174

This is not the correct colour for the background that `ArticleTitle` sits on in liveblogs:

https://github.com/guardian/dotcom-rendering/blob/ffce9fda6710ea915d59b42ac16a09574aa36d29/dotcom-rendering/src/palette.ts#L44